SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -1218,6 +1214,13 @@ void USpatialNetDriver::TickDispatch(float DeltaTime)
 	{
 		TArray<Worker_OpList*> OpLists = Connection->GetOpList();
 
+		// Servers will queue ops at startup until we've extracted necessary information from the op stream
+		if (!bIsReadyToStart)
+		{
+			HandleStartupOpQueueing(OpLists);
+			return;
```

Comment:
Is it expected behaviour that SpatialMetrics->TickMetrics() will not be called if the execution flow returns here?

Reply:
yes, I thought this would be ok for a few seconds at startup.  Are you ok with that approach?  If not we can look at adjusting things here, in a subsequent PR if that's ok

Reply:
It's best to address it in this PR or at least make sure we don't forget about it (maybe create a Jira ticket for this issue).

Reply:
As I thought of it and spoke with @m-samiec I believe it should be fine as is :)

Reply:
Alex and I chatted offline about this, resolving